### PR TITLE
add notification on copy in connector details component

### LINKED ISSUE #24: Lack of Acknowledgement Message When Using Copy Icon
### Description - What happened? *

When users click the copy icon within the application, there is no visual or textual acknowledgement provided to indicate that the data has been successfully copied. This absence of feedback results in confusion, leading users to click the icon multiple times in uncertainty.

### Expected Behavior *

Upon clicking the copy icon, an acknowledgement message such as "Copied" should be displayed to inform the user that their action has been successfully completed.

### Observed Behavior *

The copy icon does not trigger any acknowledgement message upon being clicked. This leads to user confusion and results in multiple clicks on the icon due to the lack of confirmation.

### Steps to Reproduce

1. Launch the application and log in with valid credentials.
2. Navigate to "Connectors" and click on "Add Connectors".
3. Enter all required information and click "Next".
4. Fill in the subsequent form with valid data and click "Register".
5. On the final page, notice the presence of a copy icon. Attempt to use this icon to copy data.
6. Observe the lack of acknowledgement message upon clicking the copy icon.

### Context Information

The issue was also noticed in the "Connector" and "All Connectors" list sections. When attempting to copy information by clicking the copy icon next to any connector, the expected acknowledgement message does not appear.

### Relevant log output

_No response_

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by illfixit**
_What issues does this PR close?_
closes #24 Lack of Acknowledgement Message When Using Copy Icon

image

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: authority-portal-frontend/src/app/pages/authority-connector-detail-page/authority-connector-detail-page/authority-connector-detail-page.component.ts [hunk 1]
```diff
@@ -130,7 +131,7 @@ export class AuthorityConnectorDetailPageComponent
 
   copyToClipboard(s: string | undefined) {
     if (s) {
-      this.clipboard.copy(s);
+      this.clipboardUtils.copyToClipboard(s);
     }
```
Comment (richardtreier) [inline authority-portal-frontend/src/app/pages/authority-connector-detail-page/authority-connector-detail-page/authority-connector-detail-page.component.ts:135]: move the if into the copyToClipboard method, you could show a toast "Nothing to copy"

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,12 +17,15 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 - Fixed dashboard not reloading when switching deployment environments.
 - Cleaned up repository for open-source release:
+
```
[the remark this answers is not in the document]
Comment (illfixit) [reply CHANGELOG.md]: done

File: authority-portal-frontend/src/app/pages/authority-connector-detail-page/authority-connector-detail-page/authority-connector-detail-page.component.ts (hunk 1 above)
Comment (illfixit) [reply authority-portal-frontend/src/app/pages/authority-connector-detail-page/authority-connector-detail-page/authority-connector-detail-page.component.ts]: done

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,12 +17,15 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
...
   - Removed internal tools and utilities
   - Rebased the entire history to remove secrets and unrelated code
   - Cleaned up dead links
 - Fixed an error with input validation that prevented the use of upper-case letters in zip codes [#21](https://github.com/sovity/authority-portal/issues/21)
 - Changed order of industry select options [#22](https://github.com/sovity/authority-portal/issues/22)
```
[the remark this answers is not in the document]
Comment (illfixit) [reply CHANGELOG.md]: done

**Pull request merged**